submitter/relayer/relayer.go

```diff
@@ -487,7 +487,9 @@ func (rl *Relayer) ChainTwoTxAndSend(data1 []byte, data2 []byte) (*types.BtcTxIn
 func (rl *Relayer) buildTxWithData(data []byte, firstTx *wire.MsgTx) (*types.BtcTxInfo, error) {
 	tx := wire.NewMsgTx(wire.TxVersion)
 
-	if firstTx != nil {
+	isFirstTx := firstTx != nil
```

Comment:
should this case be called is `isSecondTx` ? i.e if the first tx is not nil then we are building second tx?